src/commons/sagas/RequestsSaga.ts

```diff
@@ -125,25 +127,22 @@ export const getUser = async (tokens: Tokens): Promise<User | null> => {
  * Will be updated after a separate db for student progress is ready
  */
 export const getAchievements = async (tokens: Tokens): Promise<AchievementItem[] | null> => {
```

Comment:
I'm not sure about the semantics of redux-saga, but I'm sure there's a reason for each request function accepting tokens separately instead of just reading from the store. That being said, do you think a reasonable default for `customFetch` is to read from the store if no tokens are passed?